configure.ac

```diff
@@ -1686,21 +1686,8 @@ X
       fi
     fi
 
-  # Warn about linking Apache with libpthread if oci8 extension is enabled on linux.
+  # Warn about Apache if oci8 extension is enabled on linux.
   if test "$PHP_OCI8" != "no"; then
-    if test "$PHP_SAPI" = "apache"; then
-      if test `uname` = "Linux"; then
-cat <<X
-+--------------------------------------------------------------------+
```

Comment:
I think we should change original to another warning message.

Reply:
Which one?